Cython/Compiler/ExprNodes.py

```diff
@@ -10218,14 +10216,25 @@ def generate_codeoj_tab_entry(self, code):
 
         argcount = len(func.args) - func.num_kwonly_args
         num_posonly_args = func.num_posonly_args  # Py3.8+ only
-        kwonlyargcount = func.num_kwonly_args
+        kwonly_argcount = func.num_kwonly_args
         nlocals = len(self.varnames.args)
         flags = '|'.join(flags) or '0'
+        first_lineno = self.pos[1]
 
-        s = (f"{filename_idx}, {argcount}, {num_posonly_args}, {kwonlyargcount}, "
-             f"{nlocals}, {flags}, {varnames_result}, {file_path_result},"
-             f"{func_name_result}, {self.pos[1]}")
-        code.putln("{%s}, /* %s */" % (s, self.result_code))
+        code.putln(
+            f"{self.result_code} = __Pyx_PyCode_New("
+            f"{argcount}, "
+            f"{num_posonly_args}, "
+            f"{kwonly_argcount}, "
+            f"{nlocals}, "
+            f"{flags}, "
+            f"{filename_idx}, "
+            f"{varnames_tuple_cname}, "
```

Comment:
varnames is before filename, and filename looks like a PyObject* rather than an index currently